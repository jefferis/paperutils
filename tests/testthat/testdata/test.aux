\relax 
\citation{Suh:2004aa}
\citation{Lewis:2015aa}
\citation{Masse:2009aa}
\citation{Masse:2009aa}
\citation{Heimbeck:2001aa,Parnas:2013aa}
\citation{Dubnau:2001aa,McGuire:2001aa,Parnas:2013aa}
\citation{Wang:2003aa}
\citation{Campbell:2010aa}
\citation{Venken:2011aa}
\select@language{american}
\@writefile{toc}{\select@language{american}}
\@writefile{lof}{\select@language{american}}
\@writefile{lot}{\select@language{american}}
\citation{Suh:2004aa,Bracker:2013aa,Faucher:2013aa}
\citation{Faucher:2013aa}
\citation{Bracker:2013aa}
\citation{Lewis:2015aa}
\citation{Aso:2014aa}
\citation{Aso:2014aa}
\citation{Aso:2014aa}
\citation{Lewis:2015aa}
\citation{Aso:2014aa,Aso:2014ab}
\citation{Lewis:2015aa}
\citation{Feinberg:2008aa}
\citation{Aso:2014ab,Owald:2015aa}
\citation{Aso:2014ab}
\citation{Owald:2015aa}
\citation{Bracker:2013aa}
\citation{Lewis:2015aa}
\citation{Lewis:2015aa}
\citation{Bouzaiane:2015aa,Owald:2015aa}
\citation{Masse:2009aa}
\citation{Ramdya:2015aa}
\citation{Wasserman:2015aa}
\citation{Fischler:2007aa}
\citation{Faucher:2013aa}
\bibstyle{1_GD_LMBD_Papers_lewisdispatch2015_currentbiology}
\bibdata{0_GD_LMBD_Papers_lewisdispatch2015_dispatchbib}
\bibcite{Suh:2004aa}{{1}{2004}{{Suh et~al.}}{{}}}
\bibcite{Lewis:2015aa}{{2}{2015}{{Lewis and Kadow}}{{}}}
\bibcite{Masse:2009aa}{{3}{2009}{{Masse et~al.}}{{}}}
\bibcite{Heimbeck:2001aa}{{4}{2001}{{Heimbeck et~al.}}{{}}}
\bibcite{Parnas:2013aa}{{5}{2013}{{Parnas et~al.}}{{}}}
\bibcite{Dubnau:2001aa}{{6}{2001}{{Dubnau et~al.}}{{}}}
\bibcite{McGuire:2001aa}{{7}{2001}{{McGuire et~al.}}{{}}}
\bibcite{Wang:2003aa}{{8}{2003}{{Wang et~al.}}{{}}}
\bibcite{Campbell:2010aa}{{9}{2010}{{Campbell and Turner}}{{}}}
\bibcite{Venken:2011aa}{{10}{2011}{{Venken et~al.}}{{}}}
\@writefile{toc}{\contentsline {section}{References}{4}}
\bibcite{Bracker:2013aa}{{11}{2013}{{Br{\"a}cker et~al.}}{{}}}
\bibcite{Faucher:2013aa}{{12}{2013}{{Faucher et~al.}}{{}}}
\bibcite{Aso:2014aa}{{13}{2014{}}{{Aso et~al.}}{{}}}
\bibcite{Aso:2014ab}{{14}{2014{}}{{Aso et~al.}}{{}}}
\bibcite{Feinberg:2008aa}{{15}{2008}{{Feinberg et~al.}}{{}}}
\bibcite{Owald:2015aa}{{16}{2015}{{Owald et~al.}}{{}}}
\bibcite{Bouzaiane:2015aa}{{17}{2015}{{Bouzaiane et~al.}}{{}}}
\bibcite{Ramdya:2015aa}{{18}{2015}{{Ramdya et~al.}}{{}}}
\bibcite{Wasserman:2015aa}{{19}{2015}{{Wasserman et~al.}}{{}}}
\bibcite{Fischler:2007aa}{{20}{2007}{{Fischler et~al.}}{{}}}
\select@language{british}
\@writefile{toc}{\select@language{british}}
\@writefile{lof}{\select@language{british}}
\@writefile{lot}{\select@language{british}}
\select@language{american}
\@writefile{toc}{\select@language{american}}
\@writefile{lof}{\select@language{american}}
\@writefile{lot}{\select@language{american}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces \textbf  {A model of context-dependent olfaction in starved flies}}}{6}}
\newlabel{fig:fruLocus}{{1}{6}}
\select@language{british}
\@writefile{toc}{\select@language{british}}
\@writefile{lof}{\select@language{british}}
\@writefile{lot}{\select@language{british}}
\select@language{american}
\@writefile{toc}{\select@language{american}}
\@writefile{lof}{\select@language{american}}
\@writefile{lot}{\select@language{american}}
\select@language{british}
\@writefile{toc}{\select@language{british}}
\@writefile{lof}{\select@language{british}}
\@writefile{lot}{\select@language{british}}
\select@language{american}
\@writefile{toc}{\select@language{american}}
\@writefile{lof}{\select@language{american}}
\@writefile{lot}{\select@language{american}}
