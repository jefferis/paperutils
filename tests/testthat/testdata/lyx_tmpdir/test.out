\BOOKMARK [1][-]{section*.1}{\376\377\000I\000n\000t\000r\000o\000d\000u\000c\000t\000i\000o\000n}{}% 1
\BOOKMARK [1][-]{section*.2}{\376\377\000R\000e\000s\000u\000l\000t\000s}{}% 2
\BOOKMARK [2][-]{section*.3}{\376\377\000R\000e\000s\000u\000l\000t\000s\000\040\000s\000e\000c\000t\000i\000o\000n\000\040\0001}{section*.2}% 3
\BOOKMARK [2][-]{section*.4}{\376\377\000R\000e\000s\000u\000l\000t\000s\000\040\000s\000e\000c\000t\000i\000o\000n\000\040\0002}{section*.2}% 4
\BOOKMARK [1][-]{section*.5}{\376\377\000D\000i\000s\000c\000u\000s\000s\000i\000o\000n}{}% 5
\BOOKMARK [1][-]{section*.6}{\376\377\000M\000e\000t\000h\000o\000d\000s}{}% 6
\BOOKMARK [2][-]{section*.7}{\376\377\000F\000l\000y\000\040\000s\000t\000o\000c\000k\000s}{section*.6}% 7
\BOOKMARK [2][-]{section*.8}{\376\377\000A\000n\000a\000l\000y\000s\000i\000s}{section*.6}% 8
\BOOKMARK [1][-]{section*.9}{\376\377\000S\000u\000p\000p\000l\000e\000m\000e\000n\000t\000a\000r\000y\000\040\000I\000n\000f\000o\000r\000m\000a\000t\000i\000o\000n}{}% 9
\BOOKMARK [1][-]{section*.10}{\376\377\000A\000c\000k\000n\000o\000w\000l\000e\000d\000g\000e\000m\000e\000n\000t\000s}{}% 10
\BOOKMARK [1][-]{section*.11}{\376\377\000A\000u\000t\000h\000o\000r\000\040\000i\000n\000f\000o\000r\000m\000a\000t\000i\000o\000n}{}% 11
\BOOKMARK [1][-]{section*.12}{\376\377\000F\000i\000g\000u\000r\000e\000s}{}% 12
\BOOKMARK [1][-]{section*.15}{\376\377\000S\000u\000p\000p\000l\000e\000m\000e\000n\000t\000a\000r\000y\000\040\000I\000n\000f\000o\000r\000m\000a\000t\000i\000o\000n}{}% 13
\BOOKMARK [2][-]{section*.16}{\376\377\000S\000u\000p\000p\000l\000e\000m\000e\000n\000t\000a\000r\000y\000\040\000F\000i\000g\000u\000r\000e\000s}{section*.15}% 14
\BOOKMARK [2][-]{section*.18}{\376\377\000S\000u\000p\000p\000l\000e\000m\000e\000n\000t\000a\000r\000y\000\040\000T\000a\000b\000l\000e\000s}{section*.15}% 15
\BOOKMARK [2][-]{section*.20}{\376\377\000S\000u\000p\000p\000l\000e\000m\000e\000n\000t\000a\000r\000y\000\040\000D\000i\000s\000c\000u\000s\000s\000i\000o\000n}{section*.15}% 16
